course/api/views.py

```diff
@@ -175,10 +176,8 @@ def send_mail(self, request, *args, **kwargs):
                 return Response(_("SEND_EMAIL_FAILED"))
 
 
-class CourseExercisesViewSet(NestedViewSetMixin,
-                             CourseModuleResourceMixin,
-                             CourseResourceMixin,
-                             viewsets.ReadOnlyModelViewSet):
+class CourseExercisesViewSet(CourseResourceMixin, viewsets.ViewSet):
```

Comment:
I would have thought that removing the super class `NestedViewSetMixin` breaks all the views that depend on nesting. The URL used to nest exercises under the course, e.g., `/api/v2/courses/COURSEID/exercises/EXERCISEID/`. Why did you do this?

`CourseModuleResourceMixin`: how could this be removed?
`viewsets.ReadOnlyModelViewSet`: this used to guarantee that the view does not allow write operations such as HTTP POST and PUT. Why would you use `viewsets.ViewSet`?